### PR TITLE
(#131) Isolated nodes will return properly

### PR DESCRIPTION
**Pull request by dzhou343**
Backend change to query and response formatting to allow for single nodes with no relations to be returned when searched for by their Id.

### CODE REVIEW

**Comment by kwannoel** [comment]
Have you tested this on the frontend @dzhou343 ?

**Comment by dzhou343** [comment]
@kwannoel Just found another bug during testing for frontend. Please ignore review until I fix this up.

**Comment by dzhou343** [comment]
@kwannoel Have amended the previous commit and added a small fix. Tested on frontend and I can render an individual person by reproducing issue (#131 )

File: backend/src/convert-search-resource-id.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 // Search resource by Id, returns associated terms, labels, relations
 const convertSearchResourceId = ({ resourceId }) => {
   const params = { resourceId }
-  const query = 'MATCH (person)-[relation]-(term) WHERE ID(person) = $resourceId RETURN person,relation,term'
+  const query = 
+    'MATCH (person)-[relation]-(term) WHERE ID(person) = $resourceId RETURN person,relation,term UNION MATCH (person),(relation),(term) WHERE ID(person) = $resourceId RETURN person, null as relation, null as term'
```
Comment (kwannoel) [inline backend/src/convert-search-resource-id.js:5]: For readability let's break `UNION MATCH ...` into a new line and the `RETURN` statement as well.

**Comment by dzhou343** [comment]
@kwannoel integration test added to test bug fix feature.

**Pull request merged**